.gitignore

```diff
@@ -4,7 +4,6 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
-# Mac OS
```

Comment:
@OliverCai0 can you remove these changes from this PR?  I don't think they're necessary